src/Globe/TileGeometry.js

```diff
@@ -213,15 +214,16 @@ TileGeometry.prototype.computeBuffers = function(params, builder) {
     }
 
     var iStart = idVertex;
-    var rmax = 5000;
-    var r = Math.max(rmax, Math.pow(rmax, 1 / params.zoom));
+
+    //TODO: WARNING beware size'skirt influence performance
+    var rmax = 50;
+    var r = Math.pow(rmax, 1 + 1.25/ params.zoom);
```

Comment:
I simplify the formula. 
Skirt's size influence performance. For the moment, I reduce the size to increase performance (pixel shader performance)
But this calculation isn't correct. A correct calculation must to minimize the size without crack between tiles. This size must be take into account the bbox's size.